ch.protonmail.protonmail-bridge.yaml

```diff
@@ -32,7 +32,7 @@ modules:
         x-checker-data:
           type: html
           url: https://protonmail.com/download/current_version_linux.json
-          version-pattern: protonmail-bridge_(\d.\d.\d)-1_amd64.deb
-          url-template: https://protonmail.com/download/protonmail-bridge_$version-1_amd64.deb
+          version-pattern: protonmail-bridge_(\d.\d.\d-\d)_amd64.deb
```

Comment:
@A6GibKm Can I suggest more flexible regexp:
 ```
protonmail-bridge_(\d+.\d+.\d+(?:-\d+)?)_amd64.deb
```

Test data:
```
protonmail-bridge_1.3.3-1_amd64.deb
protonmail-bridge_1.37.3-1_amd64.deb
protonmail-bridge_1.3.17-1_amd64.deb
protonmail-bridge_1.3.17-18_amd64.deb
protonmail-bridge_1.3.17_amd64.deb
```

Reply:
Seems ok, please open a PR, you might also consider doing like other apps which just use `[\d.]+` instead of `[\d.\d.\d.]` too.